bar/check_if_nginx_install.sh

```diff
@@ -1,12 +1,30 @@
 #!/usr/bin/bash
-$1=rpm -q
-$2=bool
-if [[ $1 && $2  ]]
-then
-    echo "$1 installed"
-    sudo  yum -y autoremove
+if [[ $? -eq 0 ]]; then
+  read -p "nginx is already installed. uninstall it? [y/n]" user_ans
+  if [[ $user_ans == "y" ]]; then
+    echo "Uninstalling nginx..."
+    systemctl stop nginx.service
+    systemctl disable nginx.service
+    userdel -r nginx
+    rm -rf /etc/nginx
```

Comment:
You can combine it into single `rm -rf` command:
~~~
rm -rf folder1/ folder2/ folder3/
~~~

Reply:
fixed